intTests/test1646/test15.log.good

```diff
@@ -0,0 +1,348 @@
+ <stdin>:1:5-1:6: Warning: Redeclaration of y
+ <stdin>:1:16-1:17: Warning: Previous declaration was here
+abc : ProofScript ()
+abstract_symbolic : Term -> Term
+add_cryptol_defs : [String] -> Simpset -> Simpset
+add_cryptol_eqs : [String] -> Simpset -> Simpset
+add_prelude_defs : [String] -> Simpset -> Simpset
```

Comment:
This test worries me a little. We'll have to update it every single time that we add a new command, and that happens fairly regularly...

Reply:
Yeah, I get you. At the same time, it's something that every contributor is going to need to remember to update going forward whenever they add a new command. I suppose I can live with that, but I suspect that it'll get old pretty fast.